documentation/modules/configuring/con-config-high-volume-messages.adoc

```diff
@@ -0,0 +1,216 @@
+// Module included in the following assemblies:
+//
+// assembly-config-high-volumes.adoc
+
+[id='con-high-volume-config-properties-{context}']
+= Handling high volumes of messages
+
+[role="_abstract"]
+Operations like tracking web activity can generate a high volume of messages. 
```

Comment:
As mentioned before, I would leave this sentence out. It is confusing from my point of view:
* No clear numbers what high volume means, so it IMHO does not contribute to any understanding
* High volumes can obviously achieved with many other use-cases, so why pick one?

It might lead readers to thing that this is the use-cases which produces the high-volume of messages for which this tuning applies.

I would leave just the next sentence which also does not include any numbers, but does not confuse anyone by mentioning one specific use-case.

Reply:
Removed. I thought it was just the other sentence you wanted to be removed.